controllers/C_Document.class.php

```diff
@@ -660,7 +660,7 @@ function queue_action_process() {
 			
 			//validate that the pod exists
 			$d = new Document($doc_id);
-			$sql = "SELECT pid from patient_data where pubpid = '" . $file['patient_id'] . "'";
```

Comment:
I think that is just "owner" or foreign_id in the documents table.  I have done this in the past and everything was cool...but that was only due to the fact that pubpid=pid.
  The conversion script would need to include that, yes...but I noticed that the Libreupgrade script was just pulled out of the code base by someone....  :)